<ui version="4.0" >
 <class>PolygonWidget</class>
 <widget class="QWidget" name="PolygonWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PolygonWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="_drawBorder" >
       <property name="text" >
        <string>Draw Border</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_drawInterior" >
       <property name="text" >
        <string>Draw Interior</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Border Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_borderWidth" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Border Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtTools::ColorButton" name="_borderColor" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="checkBox_6" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Rasterize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Fit to terrain</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header>QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
